modules/nw-anp-nodes-peer-concept.adoc

```diff
@@ -0,0 +1,232 @@
+// Module included in the following assemblies:
+//
+// * networking/network_security/AdminNetworkPolicy/ovn-k-egress-nodes-networks-peer.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="nw-anp-egress_{context}"]
+= Northbound traffic controls for AdminNetworkPolicy and BaselineAdminNetworkPolicy
+
+In addition to supporting east-west traffic controls, ANP and BANP also allow administrators to control their northbound traffic leaving the cluster or traffic leaving the node to other nodes in the cluster. End-users can do the following:
+
+* Implement egress traffic control towards cluster nodes using `nodes` egress peer.
+
+* Implement egress traffic control towards Kubernetes API servers using `nodes` or `networks` egress peers.
+
+* Implement egress traffic control towards external destinations outside the cluster using `networks` peer.
```

Comment:
```suggestion
* Implement egress traffic control towards cluster nodes using `nodes` egress peer

* Implement egress traffic control towards Kubernetes API servers using `nodes` or `networks` egress peers

* Implement egress traffic control towards destinations outside the cluster using `networks` peer
```
I believe periods are not needed for incomplete sentences, but feel free to leave them if I am wrong